### ISSUE TITLE
The default terminal size in a generated config file has columns and lines reversed

### ISSUE THREAD

**Issue by rob-smallshire** [sixty-north/gjenta#46]
If you generate a config file with `gjenta default-config` you get a config file with:

```
[gjenta.terminal]
autosize = false
size = [ 20, 80,]

[gjenta.interpreter.bash]
command = "bash"

[gjenta.interpreter.python]
command = "python"

[gjenta.renderer.robot]
interval = 0.1

[gjenta.syntax.gjenta]
argv = []

```

Conventionally terminal sizes are [columns, lines] not [lines, columns] so something is reversed somewhere.

Possible further improvement: Default xterm sizes tend to be 80x24 not 80x20.

**Comment by abingham** [comment]
This is just the result of the default value that we provide in `gjenta.config`. The description is also wrong (though consistent with the default) since it says "Size of terminal ([rows, columns])".

**Comment by abingham** [comment]
Fixed in 2a081c4141c292af0355a2ffed79edcafa53c1f9
